shared_requirements.txt

```diff
@@ -156,6 +156,7 @@ opencensus-ext-threading
 opencensus-ext-azure>=0.3.1
 opentelemetry-api==0.16b1
 opentelemetry-sdk==0.16b1
+psutil>=5.8.0
```

Comment:
do we need this in the shared_requirements?

Reply:
CI complains if not added.